be/src/exec/pipeline/pipeline_driver.h

```diff
@@ -335,6 +337,13 @@ class PipelineDriver {
     void _mark_operator_closed(OperatorPtr& op, RuntimeState* runtime_state);
     void _close_operators(RuntimeState* runtime_state);
 
+    void _yield(size_t total_chunks_moved, size_t total_rows_moved, size_t time_spent) {
```

Comment:
This function is only update the metrics, why name `_yield`?

Reply:
The reason is that it is called at all the places where the driver yields. `_yield()` means this method does something when the driver yields. Maybe it isn't precise, let's rename it to `_update_metrics()`.